resources/dicts/events/death/general.json

```diff
@@ -1541,31 +1586,30 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c couldn't miss today! Such a bright day out, a perfect day to go for a walk. That was until m_c detected another scent, a strong one, and a foul one. {PRONOUN/m_c/subject/CAP} scrambled away from the area, but not before abnormally long claws slashed across their face.",
+        "event_text": "m_c couldn't miss today! Such a bright day, perfect for a walk. That was until m_c detected another scent, a strong one, and a foul one. {PRONOUN/m_c/subject/CAP} scrambled away from the area, but not before abnormally long claws slashed across their face.",
```

Comment:
this event has always read as a little too vague for me.  can we add another sentence to it? perhaps "Later on, only blood and the scent of a predator serve as evidence of the event." and then add "no_body" to the tags, since there's no reason for a predator to leave anything behind.